pkgs/by-name/an/anytype/package.nix

```
@@ -1,42 +1,106 @@
 {
   lib,
-  fetchurl,
-  appimageTools,
-  makeWrapper,
-  nix-update-script,
+  callPackage,
+  fetchFromGitHub,
+  buildNpmPackage,
+  pkg-config,
+  libsecret,
+  electron,
+  makeDesktopItem,
+  copyDesktopItems,
   commandLineArgs ? "",
+  nix-update-script,
 }:
 
 let
+  anytype-heart = callPackage ./anytype-heart { };
   pname = "anytype";
   version = "0.44.0";
-  name = "Anytype-${version}";
-  src = fetchurl {
-    url = "https://github.com/anyproto/anytype-ts/releases/download/v${version}/${name}.AppImage";
-    hash = "sha256-+Ae0xH6ipNZgIVrrAmgeG8bibm/I3NLiDMzS+fwf9RQ=";
+
+  src = fetchFromGitHub {
+    owner = "anyproto";
+    repo = "anytype-ts";
+    tag = "v${version}";
+    hash = "sha256-a2ZnTEAFzzTb+lxtQkC6QLG5SP1+gDSjI9dqUNZWfCg=";
+  };
+  description = "P2P note-taking tool";
+
+  locales = fetchFromGitHub {
+    owner = "anyproto";
+    repo = "l10n-anytype-ts";
+    rev = "a5c81ad55383c4e6e9bb7893ecfcb879bac87bea";
+    sha256 = "sha256-evSB0ohHm++tZiazXRMR4vj34IfW3HIkfZ2gwsi/2dk=";
```

We could use `hash` here instead of `sha256`.
```suggestion
    hash = "sha256-evSB0ohHm++tZiazXRMR4vj34IfW3HIkfZ2gwsi/2dk=";
```